lib/streamlit/elements/exception_proto.py

```diff
@@ -39,10 +40,15 @@ def marshall(exception_proto, exception):
     # If this is a StreamlitAPIException, we prune all Streamlit entries
     # from the exception's stack trace.
     is_api_exception = isinstance(exception, StreamlitAPIException)
+    is_deprecation_exception = isinstance(exception, StreamlitDeprecationWarning)
     is_markdown_exception = isinstance(exception, MarkdownFormattedException)
 
-    stack_trace = _get_stack_trace_str_list(
-        exception, strip_streamlit_stack_entries=is_api_exception
+    stack_trace = (
+        []
```

Comment:
Curious, why do we empty the stack trace specifically for deprecation exceptions?

Reply:
I thought it was too much to show for a deprecation warning and cluttered it up. I figured it'd be easy to find but that could just be me. Hoping to get some product thoughts on this :)